### PR TITLE
Ubuntu 17.04

### PR DESCRIPTION
**Pull request by scopatz**
Here is a dockerfile for pyne on 17.04. Currently this relies on pyne/pyne#906.  However, after that is merged, this can be changed.

### CODE REVIEW

**Comment by scopatz** [comment]
Ok I think this is ready to go in now.

File: ubuntu_17.04.dockerfile [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+# put MOAB on the path
+ENV LD_LIBRARY_PATH $HOME/opt/moab/lib:$LD_LIBRARY_PATH
+ENV LIBRARY_PATH $HOME/opt/moab/lib:$LIBRARY_PATH
+
+# Install PyNE
+RUN cd $HOME/opt \
+    && git clone https://github.com/pyne/pyne.git \
+    && cd pyne \
+    && git checkout develop \
+    && python3 setup.py install --user -j 3 \
```
Comment (FlanFlanagan) [inline ubuntu_17.04.dockerfile]: I tried to test this in docker by ran out of memory (granted my VM only has 4.8gigs available). I thought you said that -j2 was an issue for circle ci?

Comment (scopatz) [reply ubuntu_17.04.dockerfile]: It is, but this doesn't normally run on circle.  I can remove the `-j` if you want.

Comment (scopatz) [reply ubuntu_17.04.dockerfile]: done.

**Pull request merged**